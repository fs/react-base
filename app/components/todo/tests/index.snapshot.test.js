import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Todo from '../';

describe('Todo', () => {
  let props;
  const renderComponent = () => shallow(<Todo {...props} />);

  beforeEach(() => {
    props = {
      openModal: () => {},
    };
  });

  it('renders correctly', () => {
    expect(toJson(renderComponent())).toMatchSnapshot();
  });
});
