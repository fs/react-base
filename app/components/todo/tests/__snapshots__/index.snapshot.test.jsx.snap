// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo renders correctly 1`] = `
<Grid
  bsClass="container"
  componentClass="div"
  fluid={false}
>
  <Row
    bsClass="row"
    className="show-grid"
    componentClass="div"
  >
    <Col
      bsClass="col"
      componentClass="div"
      md={8}
    >
      <h2>
        Todo List
      </h2>
    </Col>
    <Col
      bsClass="col"
      componentClass="div"
      md={4}
    >
      <Button
        active={false}
        block={false}
        bsClass="btn"
        bsStyle="primary"
        className="btn btn-primary pull-right spacingTop"
        disabled={false}
        onClick={[Function]}
      >
        New Task
      </Button>
    </Col>
  </Row>
  <Row
    bsClass="row"
    className="show-grid"
    componentClass="div"
  >
    <Col
      bsClass="col"
      componentClass="div"
      md={6}
    >
      <h3
        className="spacing-bottom"
      >
        Incomplete
      </h3>
      <Connect(TodoListContainer)
        isComplete={false}
      />
    </Col>
    <Col
      bsClass="col"
      componentClass="div"
      md={6}
    >
      <h3
        className="spacing-bottom"
      >
        Complete
      </h3>
      <Connect(TodoListContainer)
        isComplete={true}
      />
    </Col>
  </Row>
</Grid>
`;
